jbi/runner.py

```diff
@@ -68,7 +68,7 @@ def execute_action(
             ) from err
         runner_context = runner_context.update(action=action)
 
-        if bug.is_private and not action.allow_private:
+        if bug.is_private:
             raise IgnoreInvalidRequestError(
                 f"private bugs are not valid for action {action.whiteboard_tag!r}"
             )
```

Comment:
Should we move this guard to the top of this function? We probably don't even want to [try and get the bug](https://github.com/mozilla/jira-bugzilla-integration/blob/dbdbc8bb7bb0b502d761cf4c5d988ffe2fad2bcd/jbi/runner.py#L52) if it's private.

Reply:
Also, the error message should probably say something about all of JBI not processing private bugs, not just specific actions.